tests/components/select/common.py

```diff
@@ -0,0 +1,25 @@
+"""Common helpers for select entity component tests."""
+
+from homeassistant.components.select import SelectEntity
+
+from tests.common import MockEntity
+
+
+class MockSelectEntity(MockEntity, SelectEntity):
+    """Mock Select class."""
+
+    _attr_current_option = None
+
+    @property
+    def current_option(self):
+        """Return the current option of this select."""
+        return self._handle("current_option")
+
+    @property
+    def options(self) -> list:
+        """Return the list of available options of this select."""
+        return self._handle("options")
+
+    def select_option(self, option: str) -> None:
+        """Change the selected option."""
+        self._attr_current_option = option
```

Comment:
This won't work since we've overwritten `current_option`. We should probably set the option in `self._values` of the mock entity instead.